app/views/unattended/provisioning_templates/snippet/kickstart_rhsm.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+kind: snippet
+name: kickstart_rhsm
+model: ProvisioningTemplate
+snippet: true
+description: |
+  Generate Anaconda RHSM configuration for Red Hat OS registration
+test_on:
+- rhel9_dhcp
+-%>
+<%
+  subman_keys = host_param('kt_activation_keys') || host_param('activation_key')
+  subman_org = host_param('subscription_manager_org')
```

Comment:
this should have `|| @host.rhsm_organization_label` otherwise the org must be specified with `subscription_manager_org` parameter

Reply:
I think all users have this `subscription_manager_org` parameter set, otherwise they weren't able to provision rhel 8 hosts

Reply:
But I agree with the change, but I think it should be like this:

`subman_org = @host.rhsm_organization_label || host_param('subscription_manager_org')`